lib/ui/survey_question/answer/answer_multiple_choices.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:survey_flutter_ic/model/survey_answer_model.dart';
+import 'package:survey_flutter_ic/model/survey_question_model.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+
+const _defaultSelectedChoiceIndex = -1;
+
+final selectedChoicesIndexProvider = StateProvider.autoDispose<List<int>>(
+    (_) => <int>[_defaultSelectedChoiceIndex]);
```

Comment:
@Wadeewee The selected choices will be applied to the other questions from the previously selected one. Please kindly check 🙏 

https://github.com/Wadeewee/survey-flutter-ic-pooh/assets/6950766/36994c1d-d9d4-46f6-9df2-c74c6b63d5be

Reply:
Yep, @hoangnguyen92dn, This one will be solved on the integration task of the `submit` survey it happens as well on `AnswerEmojiRating`.  So, I will update `todo` to reset index like I did on AnswerEmojiRating 🙏